docs/developer/price-expiration.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Price Expiration
+sidebar_label: Price Expiration
+---
+
+
+## Introduction
+When adding a line to an order, the product variant price is taken from the relevant channel listing. 
+However, throughout the order lifecycle, price recalculations may be required, and channel listing prices can be updated in the meantime.
+To ensure price consistency over time, regardless of channel listing updates, Saleor provides a settings that allows to control the price freeze period.
+
+### Settings
+You can configure the duration after which a price is considered outdated and requires refreshing from the channel listings. 
+This is managed through the [`channel`](developer/channels/configuration.mdx) settings:
+- `draftOrderLinePriceFreezePeriod`:  Defines the time (in hours) after which the order line price will be refreshed. The default value is 24 hours. 
+To prevent price refreshing entirely, set this value to null or 0.
+- `checkoutLinePriceFreezePeriod`: To be added. Currently, each recalculation retrieves the latest price from the channel listing.
```

Comment:
This could be misleading, as you provide the `checkoutLinePriceFreezePeriod`, which does not exist. Personally, I would add a separate task about updating this section when we introduce the same flow for the checkout. 
But I would keep the sentence about checkout part: 

> Currently, each recalculation retrieves the latest price from the channel listing.

Also IMHO, would be goot to mention that this flow was added in 3.21